server/push_test.go

```diff
@@ -592,7 +594,131 @@ func TestShouldNotPanicIfSessionAlreadyRemoved(t *testing.T) {
 	refreshIntervalHttpPush = oldRI
 }
 
-func TestFileExtensionError(t *testing.T) {
+func TestPush_ResetWatchdog(t *testing.T) {
+
+	wgWait(&watchdogWg) // wait for any earlier tests to complete
```

Comment:
If `wgWait()` returns false that means we timed out for some reason before `watchDogWg.Wait()` completes right? Should we assert that it returns true since if it returns false that would imply there was a problem waiting for earlier tests to complete?

Reply:
Yep, fixed in https://github.com/livepeer/go-livepeer/commit/8c1e19495b48daeb15c6ab5fa9ba313d686472bc

For reference, this is needed to make tests race-clean, otherwise it complains when we reset the watchdogResetTimer/httpPushResetTimer function: since the reset timer is run asynchronously, an earlier test may still be in the process of "finishing up" when this test case resets the function.